### ISSUE TITLE
Log folder turning to read-only on Windows -- causing error with py manager.py view

### ISSUE THREAD

**Issue by brianwfreeman** [swar/Swar-Chia-Plot-Manager#919]
Something is making the log folder turn to read-only and then causing a failure for py manager.py view.  I tried creating a new folder for the logs and it did the same thing - make the folder read only and then started throwing errors.

**Comment by Albanox** [comment]
The same here, after it does some plots, it stops and I can't 'view' anymore![image](https://user-images.githubusercontent.com/78875198/119736184-a26a5880-be7d-11eb-8d2a-b4243f81e84c.jpeg)

**Comment by brianwfreeman** [comment]
Yep that's the same error I get. It's because the log folder is locked as
read-only.



Brian W. Freeman

Attorney at Law

Certified Specialist in CA Workers' Compensation Law



http://WorkersCompensationLawyerInc.com
<http://workerscompensationlawyerinc.com/>

Phone & Fax: (844) 4 AVG JOE

**Comment by Albanox** [comment]
I just tried to change the log folder, changed config.yaml and now I can see it, let's see for how long 🥺

**Comment by brianwfreeman** [comment]
It works for one run and then the same thing will happen again.



Brian W. Freeman

Attorney at Law

Certified Specialist in CA Workers' Compensation Law



http://WorkersCompensationLawyerInc.com
<http://workerscompensationlawyerinc.com/>

Phone & Fax: (844) 4 AVG JOE

**Comment by creamsoup** [comment]
can you list files in the log directory? is there any other logs other than plot logs?

**Comment by Albanox** [comment]
No , no other logs, created a new folder and also changed it in config file, still the same after finishing ~one plot

**Comment by Albanox** [comment]
I can see the progress until let's say 60% than it stops... it says manager stopped and the error than...

**Comment by creamsoup** [comment]
it sounds like unrelated to swar to me. but i could be wrong because  i don't really know windows well.
